jme3-core/src/main/java/com/jme3/math/FastMath.java

```diff
@@ -1135,4 +1135,13 @@ public static short convertFloatToHalf(float flt) {
     public static float unInterpolateLinear(float value, float min, float max) {
         return (value - min) / (max - min);
     }
+
+    /**
+     * Round n to a multiple of p
```

Comment:
Just wanted to leave a friendly reminder that this code does not do what it is documented as doing. For instance, try `toMultipleOf(64, 11)` -- you won't get 66, that's for sure. Even `toMultipleOf(66, 11)` returns 76... Either this has some significant restrictions on either n or p that aren't documented, or it's totally bugged.